pages/common/clang++.md

```diff
@@ -0,0 +1,25 @@
+# clang++
+
+> Compiles C++ source files.
+> Part of LLVM.
+> More information: <https://clang.llvm.org>.
+
+- Compile a source code file into an executable binary:
+
+`clang++ {{source.cpp}} -o {{output_executable}}`
```

Comment:
Change throughout
```suggestion
`clang++ {{path/to/source.cpp}} -o {{path/to/output_executable}}`
```

Reply:
Is there any reason to use this syntax here, but not in `clang`, `gcc`, or `g++`?

Reply:
Probably a rule which wasn't used at the time.

Reply:
Then we should fix this for all the other pages, too.